src/components/RollingBannerWindow/RollingBannerWindow.js

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect, useRef, useState } from 'react';
+import './RollingBannerWindow.scss';
+import windowBanner from '../../windowBanner.json';
+
+const RollingBannerWindow = () => {
+  const [refSlide, setRefSlide] = useState(0);
+  const [windowData] = useState(windowBanner);
+  const slideRef = useRef(null);
+
+  useEffect(() => {
+    slideRef.current.style.transform = `translateX(-${refSlide}00%)`;
+  }, [refSlide]);
+
+  const moveDotSlide = index => {
+    setRefSlide(index);
+  };
+
+  useEffect(() => {
+    const timer = setInterval(() => {
+      setRefSlide(prev => prev + 1);
+      if (refSlide === 1) {
+        setRefSlide(0);
+      }
+    }, 4000);
+    return () => clearInterval(timer);
+  });
+
+  return (
+    <div className="rollingBannerWindow">
+      <section className="rollingWindowContainer" ref={slideRef}>
+        {windowData.map(img => {
+          return (
+            <div id="windowCard" key={img.id}>
+              <img src={img.src} alt="피드 이미지" className="imgScss" />
+              <ul className="textSet">
+                <li className="content">{img.content}</li>
+                <li className="sub">{img.sub}</li>
```

Comment:
```suggestion
        {windowData.map(img => {
        const {id, src, content, sub} = img
          return (
            <div id="windowCard" key={id}>
              <img src={src} alt="피드 이미지" className="imgScss" />
              <ul className="textSet">
                <li className="content">{content}</li>
                <li className="sub">{sub}</li>
```
이 부분도 위와같이 구조분해 할당이 가능합니다~